tests/unit/test_dbapi_cursor.py

```diff
@@ -601,3 +601,9 @@ def test__format_operation_w_too_short_sequence(self):
             "SELECT %s, %s;",
             ("hello",),
         )
+
+    def test__format_operation_w_empty_dict(self):
+        from google.cloud.bigquery.dbapi import cursor
+
+        formatted_operation = cursor._format_operation("SELECT 1", {})
+        self.assertEqual(formatted_operation, "SELECT 1")
```

Comment:
Actually, let's put a `%` sign in this string to catch the reported error

```suggestion
        self.assertEqual(formatted_operation, "SELECT '%f'")
```